### DISCUSSION TITLE
.gitignore /public/build/

### DISCUSSION THREAD

**Discussion by freshface** [studio1902/statamic-peak#240] [Ideas]
Wouldn't it be better to not ignore this?
If you need to build this on the server all node_modules are required and take a lot of space on the server.

https://github.com/studio1902/statamic-peak/blob/c1ebad3a23d8265314e28311201ae962904919f0/.gitignore#L18

**Comment by robdekort** [comment]
Hey! I switched this around multiple times. I'm open to change it. The annoying this is that I always build on the server and if I create a new project and forget to uncomment those lines I get into git hell and have to git remove them all to prevent git conflicts.

**Reply by robdekort** [reply]
Curious what others think!

**Reply by jelleroorda** [reply]
Yup, personally I’d be all for keeping it out of git as well. It always gives you merge conflicts when working with separate branches.

**Reply by sjclark** [reply]
Definitely keep it out of GIT, it's a pain every time you run into `error: Your local changes to the following files would be overwritten by merge` (definitely didn't have that handy because I've run into it twice today already 😵‍💫).  If someone has a different workflow then its very easy for them to tweak a .gitignore

**Reply by joshuablum** [reply]
+1 for keeping it out.

**Comment by robdekort** [comment]
Ok, lets keep it like it is. Sorry @freshface.
